flipperdevices/flipperzero-firmware
# [FL-2773] Fix crash after cancelling Learn New Remote

# What's new

- Fix crash after cancelling Learn New Remote

# Verification 

1. Go to `Infrared -> Learn New Remote`, wait a couple of seconds.
2. Go back without learning a remote.
3. Go to `Saved Remotes`, open any file and send a signal.

The application should not crash.

# Checklist (For Reviewer)
